tests/e2e/vsan_test.go

```diff
@@ -63,6 +64,10 @@ func (s *VsanTestSuite) SetUpTest(c *C) {
 }
 
 func (s *VsanTestSuite) TearDownTest(c *C) {
+	if len(s.containerList) > 0 {
+		out, err := dockercli.RemoveAllContainers(s.config.DockerHosts[0])
```

Comment:
I will be removing ```dockercli.RemoveAllContainers``` function and will only remove the containers that test created.